bgcval2/download_from_mass.py

```diff
@@ -483,8 +495,25 @@ def download_from_mass(
     if auto_download:
         shared_file_path = os.path.join(paths.shared_mass_scripts, os.path.basename(download_script_path))
         print('writing file in shared path', shared_file_path)
-        shutil.copy(download_script_path, shared_file_path)
 
+        # check destination permissions:
+        i_can_write_this = os.access(shared_file_path, os.W_OK)
+        if i_can_write_this:
+            # I created this file and I own it:
+            # make local copy group writable:
+            os.chmod(download_script_path, st.st_mode | stat.S_IWGRP)
```

Comment:
This line is wrong. 
It produces the permissions: `-rwxrwsr-x`
but I want it to produce: `-rw-rw-r--`